### PR TITLE
Rename Cairo namespaces (PascalCase)

### PR DESCRIPTION
**Pull request by krebernisak**

### CODE REVIEW

File: contracts/src/chainlink/cairo/access/SimpleReadAccessController/simple_read_access_controller.cairo [hunk 1]
```diff
@@ -5,13 +5,13 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from starkware.cairo.common.bool import TRUE, FALSE
```
Comment (qperrot) [inline contracts/src/chainlink/cairo/access/SimpleReadAccessController/simple_read_access_controller.cairo]: Don't need it `TRUE` or `FALSE` they are never used 
Do we realy need `SignatureBuiltin` ?

File: contracts/src/chainlink/cairo/access/SimpleWriteAccessController/simple_write_access_controller.cairo [hunk 2]
```diff
@@ -3,16 +3,14 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
```
Comment (qperrot) [inline contracts/src/chainlink/cairo/access/SimpleWriteAccessController/simple_write_access_controller.cairo]: Do we need `SignatureBuiltin`?

File: contracts/src/chainlink/cairo/access/SimpleWriteAccessController/simple_write_access_controller.cairo [hunk 3]
```diff
@@ -3,16 +3,14 @@
...
 from starkware.cairo.common.bool import TRUE, FALSE
 
-from chainlink.cairo.access.SimpleWriteAccessController.library import (
-    simple_write_access_controller,
-)
+from chainlink.cairo.access.SimpleWriteAccessController.library import SimpleWriteAccessController
 from chainlink.cairo.access.IAccessController import IAccessController
```
Comment (qperrot) [inline contracts/src/chainlink/cairo/access/SimpleWriteAccessController/simple_write_access_controller.cairo]: Why do we import `IAccessController`?

File: contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo [hunk 4]
```diff
@@ -13,7 +13,7 @@ from starkware.cairo.common.bool import TRUE, FALSE
```
Comment (qperrot) [inline contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo:12]: Don't need `get_tx_info`

File: contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo [hunk 5]
```diff
@@ -69,7 +69,7 @@ end
```
Comment (qperrot) [inline contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo:68]: About storage_var here we are naming them with `s_` and in aggregator.cairo we aren't using the `s_`but we do add `_`at the and of the name like `config_count_` do we have to rename it or is it ok ?

Comment (krebernisak) [reply contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo:68]: Yup, definitely an inconsistency I'm looking to solve. It's part of a bigger effort in the context of this GH Issue:  https://github.com/smartcontractkit/chainlink-starknet/issues/80

For this PR I'm looking to just get the namespaces renamed.

File: contracts/src/chainlink/cairo/access/SimpleWriteAccessController/simple_write_access_controller.cairo (hunk 3 above)
Comment (krebernisak) [reply contracts/src/chainlink/cairo/access/SimpleWriteAccessController/simple_write_access_controller.cairo]: Doesn't seem we need it - removed.

**Pull request merged**